misc/RELEASE_CHECKLIST_TEMPLATE.md

```diff
@@ -0,0 +1,83 @@
+# Release Checklist
+
+## Communication
+- [ ] Create a new issue and copy&past this checklist into it (Yeah! First Step done!)
+- [ ] Talk to the team: Who should make the release?
+- [ ] Announce a "Code-Freeze". No new Pull-Request until the release is done!
+- [ ] Checkout if we have MAJOR or MINOR changes. If not we do a PATCH release.
+- [ ] The new version will be: `vX.X.Y`
```

Comment:
Strictly speaking it should be `vX.Y.Z` (so major and minor versions can be different)